### PR TITLE
Add Agisoft PhotoScan

### PR DESCRIPTION
**Pull request by waynegraham**
Adds Agisoft's Photoscan to the casks

### CODE REVIEW

File: Casks/photoscan.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'photoscan' do
+  version '1.1.0'
+  sha256 '776d2c1113ba20a3f4293bf04dbdfb971916f31b3ec7f9972778d776a475f689'
+
+  url "http://download.agisoft.com/photoscan_1_1_0.dmg"
```
Comment (vitorgalvao) [inline Casks/photoscan.rb]: Double quotes are reserved for cases where string interpolation is needed. When not using string interpolation, these should be single quotes.

File: Casks/photoscan.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#  url "http://download.agisoft.com/photoscan_#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/photoscan.rb]: However, I see you have this ready. To make this work, change `#{version}` to `#{version.gsub('.','_')}`.

Comment (waynegraham) [inline Casks/photoscan.rb]: Oops...was debugging the URL and forgot to put that back...

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)? Please @mention me when you do.

**Comment by waynegraham** [comment]
@vitorgalvao should be squashed now

**Pull request merged**